charts/pega/charts/installer/config/setupDatabase.properties.tmpl

```diff
@@ -71,4 +71,14 @@ prbootstrap.config=/opt/pega/kit/scripts/prbootstrap.properties
 prlogging.config=/opt/pega/kit/scripts/prlog4j2.xml
 
 # Create schema if absent flag - Only from Docker related deployments
-pega.schema.autocreate=true
\ No newline at end of file
+pega.schema.autocreate=true
+{{- if isTrue .Env.BYPASS_LOAD_ENGINE_CLASSES }}
```

Comment:
Do these entries actually need to be conditionalized?  Can they be included in setupDatabase.properties regardless of the values?

Reply:
I felt they need not exist.